elegant-emacs-dark-theme.el

```diff
@@ -0,0 +1,628 @@
+(deftheme elegant-emacs-dark
+  "Created 2020-08-03.")
+;;; -------------------------------------------------------------------
+
+
+;;; Font and frame size
+;;; -------------------------------------------------------------------
+(setq default-frame-alist
+      (append (list '(width  . 72) '(height . 40)
+                    '(vertical-scroll-bars . nil)
+                    '(internal-border-width . 24)
+                    '(font . "Roboto Mono Light 14"))))
+(set-frame-parameter (selected-frame)
+                     'internal-border-width 24)
+;;; -------------------------------------------------------------------
+
+
+;;; Line cursor and no blink
+;;; -------------------------------------------------------------------
+(set-default 'cursor-type  '(bar . 1))
+(blink-cursor-mode 0)
+;;; -------------------------------------------------------------------
+
+
+;;; No sound
+;;; -------------------------------------------------------------------
+(setq visible-bell t)
+(setq ring-bell-function 'ignore)
+;;; -------------------------------------------------------------------
+
+
+;;; No Tooltips
+;;; -------------------------------------------------------------------
+(tooltip-mode 0)
+;;; -------------------------------------------------------------------
+
+
+;;; Paren mode is part of the theme
+;;; -------------------------------------------------------------------
+(show-paren-mode t)
+;;; -------------------------------------------------------------------
+
+
+;;; No fringe but nice glyphs for truncated and wrapped lines
+;;; -------------------------------------------------------------------
+(fringe-mode '(0 . 0))
+(defface fallback '((t :family "Fira Code Light"
+                       :inherit 'face-faded)) "Fallback")
+(set-display-table-slot standard-display-table 'truncation
+                        (make-glyph-code ?… 'fallback))
+(set-display-table-slot standard-display-table 'wrap
+                        (make-glyph-code ?↩ 'fallback))
+(set-display-table-slot standard-display-table 'selective-display
+                        (string-to-vector " …"))
+;;; -------------------------------------------------------------------
+
+
+;;; When we set a face, we take care of removing any previous settings
+;;; -------------------------------------------------------------------
+(defun set-face (face style)
+  "Reset a FACE and make it inherit STYLE."
+  (set-face-attribute face nil
+   :foreground 'unspecified :background 'unspecified
+   :family     'unspecified :slant      'unspecified
+   :weight     'unspecified :height     'unspecified
+   :underline  'unspecified :overline   'unspecified
+   :box        'unspecified :inherit    style))
+;;; -------------------------------------------------------------------
+
+
+;;; A theme is fully defined by these six faces
+;;; -------------------------------------------------------------------
+(defgroup elegance nil
+  "Faces for the elegance theme"
+  :prefix "face-"
```

Comment:
Shouldn't we used "elegance-" or "elegance-face" instead of "face-" ?

Reply:
I did not change that but sure, no problem. For me before installing Roboto Mono it did revert to a default font but also gave a split screen error. We could create a "elegant-emacs-common.el" like Spacemacs themes and Domm ones do but we'd then still have 2 separate files and the need to add elegant-emacs-pkg.el telling the fetchers what to do with the 3 files. I saw multiple people publishing the versions separately. Doing the way I did you can also enable them from the init or M-x using `load-theme` that would get tricky having only 1 file

Reply:
I see. My concern is that if we make change to the light theme, this needs to be added (manually) to the dark theme as well.

Reply:
But I can live with that.

Reply:
Yes I see. I actually did everything on the light version then pasted it in the dark one only leaving the `;;; General colors` different. I'll shift things around a bit and if they work common.el it is!

Reply:
Ok, I'll wait for your go before merging.